src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -159,6 +162,12 @@ public async Task InitAsync()
                     throw new SubscriptionInvalidStateException($"Subscription with ID '{SubscriptionId}' cannot be opened because it requires the protocol to support Counter Includes.");
             }
 
+            if (_supportedFeatures.Subscription.TimeSeriesIncludes == false)
+            {
+                if (Subscription.TimeSeriesIncludes != null && Subscription.TimeSeriesIncludes.TimeSeries.Count > 0)
+                    throw new SubscriptionInvalidStateException($"Subscription with ID '{SubscriptionId}' cannot be opened because it requires the protocol to support TimeSeries Includes.");
```

Comment:
This is if an old client try to connect, right? What will happened here?

Reply:
the worker will be disconnected